docs/docs/components/textinput.md

```diff
@@ -24,8 +24,9 @@ autoCapitalize: 'none' | 'sentences' | 'words' | 'characters';
 // Should auto-correction be applied to contents?
 autoCorrect: boolean = true;
 
-// Should focus be applied to text input on componentDidMount?
-autoFocus: boolean = false;
+// Should be focused when the component is mounted,
+// see https://microsoft.github.io/reactxp/docs/apis/focusutils.html
+autoFocus: AutoFocus | AutoFocus[] = AutoFocus.No;
```

Comment:
See note above. Let's make this an array, not a complex type that's either an array or a scalar.